rollup.config.js

```diff
@@ -1,3 +1,22 @@
+/**
+ * Custom Rollup plugin for `import.meta.url` transformation to commonjs.
+ * The default transformation ('file:' + __filename) does not check characters in __filename,
+ * and thus can produce invalid URLs, like in https://github.com/eslint/eslint/issues/15766
+ * See https://github.com/eslint/eslint/issues/15766#issuecomment-1093941321
+ * @returns {Object} Rollup plugin object.
+ */
+function importMetaURLPlugin() {
+    return {
+        name: "custom-import-meta-url",
+        resolveImportMeta(property) {
+            if (property === "url") {
+                return "require('url').pathToFileURL(__filename)";
```

Comment:
just copied from https://www.npmjs.com/package/babel-plugin-transform-import-meta :)
```suggestion
                return "require('url').pathToFileURL(__filename).toString()";
```